app/controllers/cards_controller.rb

```diff
@@ -44,6 +44,7 @@ def set_card
   end
 
   def card_params
-    params.require(:card).permit(:original_text, :translated_text, :review_date)
+    params.require(:card).permit(:id, :original_text, 
+    	                           :translated_text, :review_date)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.